include/seqan/basic/allocator_interface.h

```diff
@@ -204,6 +204,7 @@ allocate(T const &,
     if (data)
         SEQAN_PROADD(SEQAN_PROMEMORY, count * sizeof(TValue));
 #endif
+	memset((void*)data, 0, count * sizeof(TValue));
```

Comment:
Okay, we will investigate this issue. But for now, please remove the calls to memset from this PR. Thanks!